kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -424,4 +480,18 @@
     margin-left: 8px;
   }
 
+  .view-all-text {
+    padding: 60px;
+    margin-left: 60px;
+    font-size: 20px;
+    font-weight: bold;
+    text-align: center;
+  }
+
+  .view-all-card {
+    width: 370px;
```

Comment:
This width is overriding the use of the grid. You can remove it. I do think it would be good to add a _height_ however - the max height of the cards in figma is 80px. This should be updated here and in the `UnpinnedDevice` cards.